development/schema_validate.py

```diff
@@ -6,7 +6,7 @@
 # TODO: Rather nasty hack, when volatility's actually installed this would be unnecessary
 sys.path += ".."
 
-import logging
+import logging  # noqa: E402
```

Comment:
I'd move this import up?  I don't *think* it need to be below the `sys.path` modification, that's just for importing volatility3 below (which will sadly still need the noqa).  5:S

Reply:
fixed here: https://github.com/volatilityfoundation/volatility3/pull/1364/commits/436cc30fbc9dba31f988a0c786764cbb369a2d70